src/matching/Generic_vs_generic.ml

```diff
@@ -1797,7 +1797,11 @@ and m_call_op aop toka aargs bop tokb bargs tin =
         tin.config.commutative_boolop
     | G.Eq
     | G.NotEq ->
-        tin.config.commutative_compop
+        if tin.config.commutative_compop then
+          logger#error
```

Comment:
Note that `logger#error` will likely not be seen by any user, so it isn't very useful. You would need to report this as an error that is then passed to the CLI, or just check for this in the CLI.

Reply:
It could be fine to just leave the note in the changelog.